src/vstart.sh

```diff
@@ -695,14 +695,14 @@ EOF
         fi
 	MGR_PORT=$(($MGR_PORT + 1000))
 
+	DASH_V2_PORT=$DASH_V2_PORT
```

Comment:
is this line doing anything?

Reply:
Without that line the DASH_V2_PORT variable was always an empty string. I think this is because this is inside another function start_mgr()

Reply:
`DASH_V2_PORT` should per default be set to `$MGR_PORT` here.

Reply:
Indeed, will fix this